doc/release-notes-24494.md

```diff
@@ -0,0 +1,8 @@
+
+To help prevent fingerprinting transactions created by the Bitcoin Core wallet, the default change
+target (MIN\_CHANGE or 1,000,000sat) has been removed. When creating a transaction, if a change
+output is to be created, the wallet now picks a random amount to target within a range based on the
+average payment value:
```

Comment:
I think users that are interested in the exact algorithm can read the source code. Otherwise this will just bloat the release notes.